opentelemetry-exporter-google-cloud/tests/test_cloud_trace_exporter.py

```diff
@@ -84,6 +84,76 @@ def setUpClass(cls):
                 "int_key": AttributeValue(int_value=123),
             }
         )
+        cls.attributes_labels_mapping = {
+            "exception.message": "exception message",
+            "exception.type": "ValueError",
+            "exception.stacktrace": "exception stacktrace",
+            "http.scheme": "http",
+            "http.host": "172.19.0.4:8000",
+            "http.method": "POST",
+            "http.request_content_length": 321,
+            "http.response_content_length": 123,
+            "http.route": "/fuzzy/search",
+            "http.status_code": 200,
+            "http.url": "http://172.19.0.4:8000/fuzzy/search",
+            "http.user_agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36",
+            "process.pid": 1111,
+        }
+        cls.extracted_attributes_labels_mapping = ProtoSpan.Attributes(
```

Comment:
nit since these are only used in the `test_extract_label_mapping_attributes` case, I'd just instantiate them there for clarity

Reply:
done